tox.ini

```diff
@@ -2,7 +2,7 @@
 name = neurom
 testdeps =
     mock
-    nose
```

Comment:
holy smokes that was fast

Reply:
There are still problem with tests locations, I have to move them to `tests` folder for simplicity